docs/root/modules/aws/schema.md

```diff
@@ -2095,6 +2095,32 @@ Representation of an AWS S3 [Bucket](https://docs.aws.amazon.com/AmazonS3/latest
         (S3Bucket)-[TAGGED]->(AWSTag)
         ```
 
+### S3PolicyStatement
+
+Representation of an AWS S3 [Bucket Policy Statements](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-policies.html) for controlling ownership of objects and ACLs of the bucket.
+
+| Field | Description |
+|-------|-------------|
+| firstseen| Timestamp of when a sync job first discovered this node  |
+| lastupdated |  Timestamp of the last time the node was updated |
+| policy_id | Optional string "Id" for the bucket's policy |
+| policy_version| Version of the bucket's policy |
+| **id** | The unique identifier for a bucket policy statement. <br>If the statement has an Sid the id will be calculated as _S3Bucket.id_/policy_statement/_index of statement in statement/_Sid_. <br>If the statement has no Sid the id will be calculated as  _S3Bucket.id_/policy_statement/_index of statement in statement  |
```

Comment:
Check the markdown rendering of `statement/_Sid_`. It italicized, which I don't think you intended.

Reply:
ah i think i just missed the italics on the index of statement! thanks